doc/src/region.rst

```diff
@@ -210,9 +210,11 @@ equal-style :doc:`variable <variable>`.  Likewise, for style *sphere*
 and *ellipsoid* the x-, y-, and z- coordinates of the center of the
 sphere/ellipsoid can be specified as an equal-style variable.  And for
 style *cylinder* the two center positions c1 and c2 for the location of
-the cylinder axes can be specified as a equal-style variable.  For style
-*cone* all properties can be defined via equal-style variables.  For
-style *plane* the point can be defined via equal-style variables.
+the cylinder axes can be specified as a equal-style variable.  For styles
+*cone* and *plane* all properties can be defined via equal-style variables.
+For style *plane*, there is the restriction that all three components
```

Comment:
This statement is not as clear as the error message in the region_plane.cpp code.
So make it say either all constants or all defined by e-style variables.